apps/NewsFeedReader/qml/feedView.qml

```diff
@@ -0,0 +1,30 @@
+import QtQuick
+import QtQuick.Controls
+
+Item {
+    id: feed
+
+    Rectangle{
+        anchors.fill: parent
+        Text {
+            id: test
+            text: qsTr("Look at all my fancy Feeds")
+            anchors.centerIn: parent
+        }
+
```

Comment:
So this QML Stackview element is the place supposed to contain the XMLListModel, is that correct?
In order for me to implement this I need your changes in my branch. So either
1.) let's merge in main (and then I create a new branch from there)
2.) I create a new branch based on your branch StackView (which would it make a little more complicated 🤣 )

Reply:
Exactly.
Ok, then I'll finish the draft so we can merge in main. No need to make it more complicated 😄